source/install/troubleshooting.rst

```diff
@@ -60,7 +60,12 @@ Locked out of System Administrator account
 
 If email sign-in was turned off before the System Administrator switched sign-in methods, sign up for a new account and promote it to System Administrator from the command line.
 
-YouTube Videos show a "Video not found" preview
+Unable to play audio or video files on browser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your server may be mis-configured where the `MIME type <https://golang.org/pkg/mime/#TypeByExtension>`_ isn't set correctly. Try installing a media player or read the `related documentation <https://golang.org/pkg/mime/#TypeByExtension>`_ to add the missing mime types.
+
+YouTube videos show a "Video not found" preview
```

Comment:
1. "may be mis-configured" -> "might be misconfigured"
2. Where would the media player be installed? This doc is aimed at admins who are troubleshooting the installation. Are the end users installing media players? Do you want the admin to install a media player on the server?
3. The two links are identical. If that's intentional, then maybe remove the first instance.
4. I'm not sure how a link into the Go programming docs helps an admin installing Mattermost. It would be more helpful to list applicable MIME types here, and/or describe how to change the MIME type settings on the server.

Maybe  this: "The server might not be using the correct MIME type for the media files. To configure MIME types on your server, see \<link-to-relevant-info\>"